lib/services/data/budget/providers/cur_org_available_budget_items.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:seren_ai_flutter/services/data/budget/models/budget_item_ref_model.dart';
+import 'package:seren_ai_flutter/services/data/budget/repositories/budget_item_refs_repository.dart';
+import 'package:seren_ai_flutter/services/data/orgs/providers/cur_selected_org_id_notifier.dart';
+
+/// This provider is used to get the available budget items for the current organization.
+/// It is used to populate the budget items autofill/autocomplete in the task budget fields.
+final curOrgAvailableBudgetItemsStreamProvider =
```

Comment:
shouldn't the name of this file contain provider to be consistent with existing naming?